config.py

```diff
@@ -8,6 +8,10 @@ class Config(object):
     SQLALCHEMY_DATABASE_URI = os.getenv('db_url')
     CSRF_ENABLED = True
     SECRET_KEY = os.getenv('secret')
+    SWAGGER_UI_DOC_EXPANSION = 'list'
```

Comment:
This commit deals with swaggerui configurations, how comes other files have been changed?